azure-pipelines-official.yml

```diff
@@ -26,22 +26,16 @@ jobs:
   steps:
   - task: NuGetToolInstaller@0
     inputs:
-      versionSpec: '4.9.2' 
-
-  - task: NuGetCommand@2
```

Comment:
This is no longer necessary. We restore packages to repo-local `.packages` directory in official build.